### PR TITLE
Add Swift examples to IAM customization page

### PR DESCRIPTION
**Pull request by Bucimis**

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 1]
```diff
@@ -275,11 +292,25 @@ func logInAppMessageClickedWithButtonID(buttonId: NSInteger)
 ## Customizing In-App Message Behavior on Click
 The `inAppMessageClickActionType` property on the `ABKInAppMessage` defines the action behavior after the in-app message is clicked. This property is read only. If you want to change the in-app message's click behavior, you can call the following method on `ABKInAppMessage`:
 
+{% tabs %}
+{% tab OBJECTIVE-C %}
+
 ```objc
 - (void)setInAppMessageClickAction:(ABKInAppMessageClickActionType)clickActionType
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: We should probably edit this to be
```
[inAppMessage setInAppMessageClickAction:clickActionType withURI:URL];
```

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 2]
```diff
@@ -275,11 +292,25 @@ func logInAppMessageClickedWithButtonID(buttonId: NSInteger)
...
                            withURI:(NSURL *)uri;
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+// sample usage
+inAppMessage.setInAppMessageClickAction(clickActionType: ABKInAppMessageClickActionType, withURI: URL?)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: `ABKInAppMessageClickActionType` is not an actual param. How about `clickActionType`?

Alternatively we can make this format match the ObjC one where it is just the method signature

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 3]
```diff
@@ -275,11 +292,25 @@ func logInAppMessageClickedWithButtonID(buttonId: NSInteger)
...
 ```objc
 - (void)setInAppMessageClickAction:(ABKInAppMessageClickActionType)clickActionType
                            withURI:(NSURL *)uri;
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+// sample usage
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: I don't think this comment is necessary

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: removed

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md (hunk 1 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: changed

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md (hunk 2 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: changed - please check the update to URL too

File: _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md [hunk 4]
```diff
@@ -275,11 +292,23 @@ func logInAppMessageClickedWithButtonID(buttonId: NSInteger)
...
-- (void)setInAppMessageClickAction:(ABKInAppMessageClickActionType)clickActionType
-                           withURI:(NSURL *)uri;
+[inAppMessage setInAppMessageClickAction:clickActionType withURI:URL];
 ```
 
+{% endtab %}
+{% tab swift %}
+
+```swift
+inAppMessage.setInAppMessageClickAction(clickActionType: clickActionType, withURI: uri)
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: nit: This line we use `uri` and in the Swift example it's `URL`

Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/ios/in-app_messaging/customization.md]: updated both to uri

**Pull request merged**